src/Command/Pull/PullCommandBase.php

```diff
@@ -244,6 +249,11 @@ protected function downloadDatabaseDump(
   ): string {
     $database_backups = new DatabaseBackups($acquia_cloud_client);
     $backups_response = $database_backups->getAll($environment->uuid, $database->name);
+    if (!count($backups_response)) {
+      $this->logger->warning('No existing backups found, creating an on-demand backup now. This will take some time depending on the size of the database.');
+      $this->createBackup($environment, $database, $acquia_cloud_client);
+      $backups_response = $database_backups->getAll($environment->uuid, $database->name);
+    }
     $backup_response = $backups_response[0];
     $this->logger->debug('Using database backup generated at ' . $backup_response->completedAt);
```

Comment:
Seems like we should add the backup ID and URL.

Reply:
Backups have no UUID, but we can print the ID.